include/acl_selectors.php

```diff
@@ -613,9 +626,9 @@ function acl_lookup(App $a, $out_type = 'json') {
 	if ($conv_id) {
 		/* if $conv_id is set, get unknow contacts in thread */
 		/* but first get know contacts url to filter them out */
-		function _contact_link($i){ return dbesc($i['link']); }
-		$known_contacts = array_map(_contact_link, $contacts);
-		$unknow_contacts=array();
+		function _contact_link($i) { return dbesc($i['link']); }
+		$known_contacts = array_map('_contact_link', $contacts);
```

Comment:
This is true, you don't have to do it now. For future reference, it would look like that:
````php
$known_contacts = array_map(function ($i) {
	return dbesc($i['link']);
}, $contacts);
````
The only thing I removed is the name of the function, and I moved it directly in the parameter list